app/src/main/java/cm/aptoide/pt/home/apps/StandByUpdateAppViewHolder.java

```diff
@@ -43,12 +50,39 @@ public StandByUpdateAppViewHolder(View itemView, PublishSubject<AppClick> cancel
 
     if (((UpdateApp) app).isIndeterminate()) {
       progressBar.setIndeterminate(true);
+      downloadInteractButtonsLayout.setVisibility(View.GONE);
+
+      LinearLayout.LayoutParams appInfoParams =
+          (LinearLayout.LayoutParams) downloadAppInfoLayout.getLayoutParams();
+      appInfoParams.weight = 3;
+      appInfoParams.rightMargin = 56;
+      downloadAppInfoLayout.setLayoutParams(appInfoParams);
+
+      LinearLayout.LayoutParams buttonsLayoutParams =
+          (LinearLayout.LayoutParams) downloadInteractButtonsLayout.getLayoutParams();
+      buttonsLayoutParams.weight = 0;
+      downloadInteractButtonsLayout.setLayoutParams(buttonsLayoutParams);
+
       cancelButton.setVisibility(View.GONE);
       resumeButton.setVisibility(View.GONE);
       updateProgress.setVisibility(View.GONE);
       updateState.setText(itemView.getResources()
           .getString(R.string.apps_short_updating));
     } else {
+
+      LinearLayout.LayoutParams appInfoParams =
+          (LinearLayout.LayoutParams) downloadAppInfoLayout.getLayoutParams();
+      appInfoParams.weight = 2;
+      appInfoParams.rightMargin = 8;
+      downloadAppInfoLayout.setLayoutParams(appInfoParams);
+
+      LinearLayout.LayoutParams buttonsLayoutParams =
+          (LinearLayout.LayoutParams) downloadInteractButtonsLayout.getLayoutParams();
+      buttonsLayoutParams.weight = 1;
+      downloadInteractButtonsLayout.setLayoutParams(buttonsLayoutParams);
+
+      downloadInteractButtonsLayout.setVisibility(View.VISIBLE);
+
```

Comment:
Duplicated code, this can be extracted and placed before the if else, or maybe to a method where you would pass the weight and the rightmargin

Reply:
done